src/lib/components/layout/v2/panels.tsx

```diff
@@ -0,0 +1,73 @@
+import { ReactElement } from 'react';
+import styled from 'styled-components';
+
+interface Ratio {
+  left: number;
+  right: number;
+}
+
+const ThirtySeventy: Ratio = {
+  left: 0.3,
+  right: 0.7,
+};
+
+const SeventyThirty: Ratio = {
+  left: 0.7,
+  right: 0.3,
+};
+
+const FiftyFifty: Ratio = {
+  left: 0.5,
+  right: 0.5,
+};
+
+export const PanelsRatios = {
+  ThirtySeventy,
+  SeventyThirty,
+  FiftyFifty,
+};
+
+const PanelsContainer = styled.div<{
+  hasPadding?: boolean;
+}>`
+  display: flex;
+  flexdirection: row;
+  flex: 1;
+  padding: ${(props) => (props.hasPadding ? '1rem' : 'initial')};
```

Comment:
I don't think that we want to provide any padding configurability on the PanelsContainer. It should not have any padding